mkt/files/helpers.py

```diff
@@ -92,8 +92,12 @@ def extract(self):
         Will make all the directories and expand the files.
         Raises error on nasty files.
         """
+        if self.file.status == mkt.STATUS_PUBLIC:
```

Comment:
Yeah, when they are signed. I believe that's it. Unsigned files should be private.